### PR TITLE
Bug 1713416 - Add Rally Study 01 to the Glean ingestion

### PR DESCRIPTION
**Pull request by Dexterp37**
**DO NOT MERGE YET. There is a pending discussion: https://bugzilla.mozilla.org/show_bug.cgi?id=1713416**

TODO:

- [x] Check with the Rally team if they are cool with using the probe dictionary.
- [ ] ~Check with the Rally team if they are cool with the recommended app id.~ (Will be handled separately after a decision is made)

### CODE REVIEW

File: repositories.yaml [hunk 1]
```diff
@@ -611,3 +611,27 @@ applications:
...
+    branch: master
+    metrics_files:
+      - metrics.yaml
+    ping_files:
+      - pings.yaml
+    dependencies:
+      - glean-js
+    channels:
+      - v1_name: rally-study-zero-one
+        app_id: rally-study-zero-one
```
Comment (acmiyaguchi) [inline repositories.yaml:635]: I'll just note that `rally_zero_one` and `rally_study_zero_one` are close enough together to be very confusing. Why not `rally_zero_two`; is this the same study?

**Review by acmiyaguchi — APPROVED** [review]
This looks good to me, in terms of getting it scheduled into mozilla-pipeline-schemas.

File: repositories.yaml (hunk 1 above)
Comment (Dexterp37) [reply repositories.yaml:635]: The study itself is called "Rally 01", so calling it "zero two" would be even more confusing :) However, we're apparently hitting a GCP limit with this name and we're [discussing potential solutions here](https://bugzilla.mozilla.org/show_bug.cgi?id=1713416).

File: repositories.yaml [hunk 2]
```diff
@@ -613,3 +613,27 @@ applications:
...
+    encryption:
+      use_jwk: true
```
Comment (Dexterp37) [inline repositories.yaml:639]:
```suggestion
      use_jwk: true
    skip_documentation: true
```

**Pull request merged**